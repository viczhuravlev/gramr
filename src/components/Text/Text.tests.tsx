import React from 'react';
import renderer from 'react-test-renderer';

import { Text } from './index';

describe('[component] Text', () => {
  test('[Snapshot] Text', () => {
    const component = renderer.create(
      <Text>
        Steve fish muskellunge swallower triplefin blenny horsefish wolffish.
      </Text>
    );

    expect(component).toMatchSnapshot();
  });
});
