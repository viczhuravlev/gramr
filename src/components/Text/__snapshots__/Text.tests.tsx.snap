// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[component] Text [Snapshot] Text 1`] = `
<span
  className="s1qvlahm t1hb7lyd"
  data-size="m"
  style={
    Object {
      "--t1hb7lyd-0": "400",
      "--t1hb7lyd-1": "",
      "--t1hb7lyd-2": "inherit",
    }
  }
>
  Steve fish muskellunge swallower triplefin blenny horsefish wolffish.
</span>
`;
